### ISSUE TITLE
Small mistake in the example in the Readme

### ISSUE THREAD

**Issue by githubjakob** [denisraslov/react-spreadsheet-grid#22]
* `react-spreadsheet-grid` version: 1.3.1
* `React` version: 16.3.1
* `Operating System`: Ubuntu 17.10

What I did:
I tried to follow the example given in the readme and when I changed the content of a cell I got this error: 
```
TypeError: rows[rowId] is undefined
```

Suggested solution:
Is there maybe a small mistake in the readme to make the example work out of the box?
In the dummy content for the rows:
```
const rows=[
    { id: 'user1', name: 'John Doe', positionId: 'position1' },
    // and so on...
];
```
I needed to change the ids to `0, 1, 2...` to make the example work for me.

**Comment by denisraslov** [comment]
Hi!

Yes, it is definitely an error there! Thank you for the attentiveness!

We should do not
```js
rows[rowId][field] = value;
```
but
```js
const row = rows.find({ id } => id === rowId);
row[field] = value;
```
I.e. we should find a row by id.

You can make a PR if you want, I would merge it 🙂

**Comment by denisraslov** [comment]
Fixed in the **release 1.4.3**.
